boids/boids.py

```diff
@@ -10,34 +10,37 @@
 
 import random
 
-boids_x=[random.uniform(-450,50.0) for x in range(50)]
-boids_y=[random.uniform(300.0,600.0) for x in range(50)]
-boid_x_velocities=[random.uniform(0,10.0) for x in range(50)]
-boid_y_velocities=[random.uniform(-20.0,20.0) for x in range(50)]
-boids=(boids_x,boids_y,boid_x_velocities,boid_y_velocities)
+class boid:
```

Comment:
[Conventionally](https://www.python.org/dev/peps/pep-0008/#naming-conventions), class names start with a capital letter, e.g. `Boid`. Also, since this contains many of them, I would call it `Boids` to make that clear.